Example/BonMot.xcodeproj/project.pbxproj

```diff
@@ -155,6 +155,7 @@
 		CDFDF5501B51880D00A1AD42 /* NSDictionary+BONEquality.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = "NSDictionary+BONEquality.m"; sourceTree = "<group>"; };
 		CDFDF5531B518F3900A1AD42 /* BONDictionaryEqualityTestCase.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = BONDictionaryEqualityTestCase.m; sourceTree = "<group>"; };
 		DC26879CFC50F3E9302E163B /* Pods-Tests.debug.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-Tests.debug.xcconfig"; path = "Pods/Target Support Files/Pods-Tests/Pods-Tests.debug.xcconfig"; sourceTree = "<group>"; };
+		E19B7EA11BF3C517002CD3D8 /* BONSpecialGenerator.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = BONSpecialGenerator.swift; sourceTree = "<group>"; };
```

Comment:
#req this is still referencing the one from the Example project.
![screen shot 2015-11-11 at 2 22 22 pm](https://cloud.githubusercontent.com/assets/464574/11100499/a405fcc4-887f-11e5-9a09-5b684b7c4591.png)